index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Adopt A Dog</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles/styles.css">
+    </head>
+    
+    <body>
+    <div>
+        <section class="img-container">
+            <div class="logo" ><img src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo">
+                <h1><i>Seattle's premiere dog adoption agency</i></h1>
+            </div>    
+            <div class="chk-cart" style="padding-bottom: 25%"> 
+                    <!-- <svg class="scale-cart" viewBox="0 0 60 55" > -->
+                        <img src="images/cart-icon.svg" alt="Checkout-cart icon">
+                        <input type="submit" value="Checkout" class="btn">
+                        <!-- <p><a href="adopt-checkout.html">Checkout</a></p> -->
+                    <!-- </svg> -->
+            </div>
+        </section>
+
+            <nav>
+                <ul>
+                    <li><a href="">Home</a></li>
+                    <li><a href="adopt-dogs.html">Dogs</a></li>
+                    <li><a href="adopt-blog.html">Blog</a></li>
+                </ul>
+            </nav>
+        
+        <!-- Note to me re:outline HTML5 https://webdesign.tutsplus.com/articles/the-truth-about-multiple-h1-tags-in-the-html5-era-webdesign-16824 -->
+        <section class="main-section">
+            <div>
+                <img src="images/home-intro.jpg" alt="Floppy ear dog facing us, human embracing dog facing away.">
+            </div>    
+            <article>
+                <h1>Adopt A Dog</h1>
```

Comment:
title could be bigger, text below it smaller. should aim to match the styling seen in the PDF mockup